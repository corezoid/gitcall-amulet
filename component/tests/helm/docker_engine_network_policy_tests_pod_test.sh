#!/usr/bin/env bash
$CZ helm template | sed -n '/docker-engine-network-policy-tests-pod\.yaml/,/Source\:/p'
