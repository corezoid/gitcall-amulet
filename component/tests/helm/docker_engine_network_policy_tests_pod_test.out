# Source: gitcall_amulet_release/charts/gitcall_amulet/templates/docker-engine-network-policy-tests-pod.yaml
apiVersion: batch/v1
kind: Job
metadata:
  name: RELEASE-NAME-gitcall-amulet-docker-engine-network-policy-test
  annotations:
    "helm.sh/hook": test
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "gitcall_amulet"
spec:
  activeDeadlineSeconds: 180 # 3min
  ttlSecondsAfterFinished: 360 # 6min
  backoffLimit: 0
  template:
    metadata:
      labels:
        tier: "docker_server"
        http: "false"
    spec:
      restartPolicy: Never
      containers:
        - name: main
          imagePullPolicy: "IfNotPresent"
          image: docker-hub.middleware.biz/gitcall/gitcall-amulet:aVersion
          command: ["/app/network_policy_test"]
          args:
            - "-test.v"
            - "--np-allow=allow:4444"
            - "--np-deny=deny:5555"
          resources:
            limits:
              cpu: 100m
              memory: 50Mi
            requests:
              cpu: 100m
              memory: 50Mi
      imagePullSecrets:
      - name: "the-image-pull-secret"
---
# Source: gitcall_amulet_release/charts/gitcall_amulet/templates/docker-registry-network-policy-tests-pod.yaml
